jdi-light-html-tests/src/main/java/io/github/com/entities/TableData.java

```diff
@@ -0,0 +1,11 @@
+package io.github.com.entities;
+
+public class TableData {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'TableData' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4840068)